views/aggregate-flow-metadata.json

```diff
@@ -240,13 +254,18 @@
                     }
                 }
             },
-            "install_command": "pip install prefect-census",
-            "description": {
-                "summary": "A flow for triggering a Census sync run and waiting for completion.",
-                "returns": "The result of the sync run."
-            }
+            "path_containing_flow": "prefect_census/flows.py",
+            "repo_url": "https://github.com/PrefectHQ/prefect-census",
+            "slug": "run_census_sync"
         },
         "Wait for Census sync run": {
```

Comment:
Wondering if keys should have spaces in them.

Reply:
does it matter if they're going to be accessed as a string?

Reply:
I think it's safer to use the actual function name / slug as the key. One edge case could be special symbols used in the title (emojis for example).

Reply:
fair, can update